index.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+
+const app = express();
+
+const topicsRouter = require("./routes/topics");
+
+app.use(express.json());
```

Comment:
We don't need this line yet as we only need this line to access the body for a get request - it'll be needed once we get to our post request later. As its best practise to only include code that meets the need of the ticket you're working on, this should be removed in this PR